src/System.Net.Sockets/tests/FunctionalTests/LoggingTest.cs

```diff
@@ -34,14 +35,34 @@ public void EventSource_EventsRaisedAsExpected()
                     var events = new ConcurrentQueue<EventWrittenEventArgs>();
                     listener.RunWithCallback(events.Enqueue, () =>
                     {
-                        // Invoke a test that'll cause some events to be generated
+                        // Invoke several tests to execute code paths while tracing is enabled
```

Comment:
Sure, but in this case:
- I use it instead of await because the delegate is an Action, not a Func<Task>, so the lambda can't be made async (at least not without terrible consequences).  In other words, the call site is synchronous.
- I use it instead of Result because these don't return a value.
- I use it instead of Wait because Wait wraps exceptions in an AggregateException (in case the task contains multiple exceptions).

If you think that needs to be in a comment, I can add one.